ramalama/transports/oci.py

```diff
@@ -195,22 +216,35 @@ def _create_manifest(self, target, imageid, args):
             "--annotation",
             f"{annotations.AnnotationModel}=true",
             "--annotation",
-            "org.containers.type=''",
+            ociimage_car if args.type == "car" else ociimage_raw,
             "--annotation",
-            f"{annotations.AnnotationTitle}=args.SOURCE",
+            f"{annotations.AnnotationTitle}={args.SOURCE}",
             target,
             imageid,
         ]
-        run_cmd(cmd_args, stdout=None)
+        if args.dryrun:
+            dry_run(cmd_args)
+        else:
+            run_cmd(cmd_args, stdout=None)
```

Comment:
We should probably add an option to run_cmd called dryrun=False, and consolidate all of these calls.